### PR TITLE
test: Show overall coverage status

### PR DESCRIPTION
**Pull request by marusak**
This will help with storing of this value.

### CODE REVIEW

**Comment by marusak** [comment]
`Overall line coverage: 87.7%` at the end of https://cockpit-logs.us-east-1.linodeobjects.com/pull-17586-20220722-084603-d48dca62-fedora-36-devel/log which agrees with https://cockpit-logs.us-east-1.linodeobjects.com/pull-17586-20220722-084603-d48dca62-fedora-36-devel/Coverage/index.html

File: test/common/lcov.py [hunk 1]
```diff
@@ -439,14 +440,20 @@ def create_coverage_report():
...
-            subprocess.check_call(["genhtml", "--no-function-coverage",
-                                   "--prefix", os.getcwd(),
-                                   "--title", title,
-                                   "--output-dir", f"{output}/Coverage", all_file])
+            summary = subprocess.check_output(["genhtml", "--no-function-coverage",
+                                               "--prefix", os.getcwd(),
+                                               "--title", title,
+                                               "--output-dir", f"{output}/Coverage", all_file]).decode()
+
+            coverage = summary.split("\n")[-2]
```
Comment (martinpitt) [inline test/common/lcov.py:452]: That feels a bit brittle, but I suppose we'll notice at some point if this stops matching/working.

**Review by martinpitt — APPROVED** [review]
Thank you! Do you plan to parse that into our test-results.db?

**Comment by martinpitt** [comment]
Oh noes @ https://cockpit-logs.us-east-1.linodeobjects.com/pull-17586-20220722-084603-d48dca62-ubuntu-2204/log.html#123-2 -- VM boot failures again :cry: Retrying for now to unblock this PR, but adding to my TODO.

File: test/common/lcov.py (hunk 1 above)
Comment (marusak) [reply test/common/lcov.py:452]: Oh yeah, I guess I could iterate over all lines...

**Comment by marusak** [comment]
> Do you plan to parse that into our test-results.db?

Yes, see https://github.com/cockpit-project/bots/pull/3643

**Pull request merged**